home/forms/search.py

```diff
@@ -3,63 +3,22 @@
 
 from django import forms
 
+from .domain_model import DomainModel
+
 
 def get_domain_choices() -> list[tuple[str, str]]:
     """Make API call to obtain domain choices"""
-    # TODO: pull in the domains from the catalogue client
-    # facets = client.search_facets()
-    # domain_list = facets.options("domains")
-    return [
-        ("", "All Domains"),
-        ("urn:li:domain:HMCTS", "HMCTS"),
-        ("urn:li:domain:HMPPS", "HMPPS"),
-        ("urn:li:domain:HQ", "HQ"),
-        ("urn:li:domain:LAA", "LAA"),
-        ("urn:li:domain:OPG", "OPG"),
+    choices: list[tuple[str, str]] = [
+        ("", "All domains"),
```

Comment:
```suggestion
def get_domain_choices() -> list[Domain]:
    """Make API call to obtain domain choices"""
    choices: list[Domain] = [
        Domain("", "All domains"),
```
The other elements of this list are `Domain`s, so bringing this in line and increasing specificity of the typing